modules/database/src/adapters/mongoose-adapter/index.ts

```diff
@@ -30,6 +38,7 @@ export class MongooseAdapter extends DatabaseAdapter<MongooseSchema> {
     this.registeredSchemas = new Map();
     this.connectionString = connectionString;
     this.mongoose = new Mongoose();
+    this.mongoose.pluralize(null);
```

Comment:
Conduit should explicitly pluralize schema names consistently across both Mongoose and Sequelize.

This is going to require a **migration** for existing SQL deployments as their current system schema names are going to differ. Notably, `_DeclaredSchema !== _DeclaredSchemas` which will be result in the collection getting recreated ootb.
We could probably(?) avoid migrating declared schema entries by checking against the collection contents or something if we don't wish to migrate everything.